docs/guides/scalar.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Custom Scalars
+---
+
+The GraphQL specification includes default scalar types Int, Float, String, Boolean, and ID. Although these scalars cover the majority of use cases, some applications need to support other atomic data types (such as Date) or add validation to an existing type.
+Tailcall supports a set of predefined custom scalars to enhance the functionality and validation of your GraphQL APIs
+
+Currently supported scalars:
+
+- `Email`
+- `PhoneNumber`
+- `Date`
+- `Url`
+
+:::tip
+If a scalar you need is not supported, please create an issue on the Tailcall GitHub repository. 
+:::
+
+
+## Example Usage
```

Comment:
We don't need to give examples of each and every scalar.

Reply:
`JSON` is one important scalar that's missed over here. We should add some documentation about it as well.


Recommend users to use `JSON` when no scalar is applicable for your usecase.